backend/api/Areas/Documents/DocumentController.cs

```diff
@@ -38,152 +37,135 @@ public class DocumentController : ControllerBase
         /// Creates a new instance of a ErrorController class.
         /// </summary>
         /// <param name="documentService"></param>
-        /// <param name="documentSyncService"></param>
         /// <param name="mapper"></param>
-        public DocumentController(IDocumentService documentService, IDocumentSyncService documentSyncService, IMapper mapper)
+        public DocumentController(IDocumentService documentService, IMapper mapper)
         {
             _documentService = documentService;
-            _documentSyncService = documentSyncService;
             _mapper = mapper;
         }
         #endregion
 
         #region Endpoints
 
         /// <summary>
-        /// Retrieves the list of document types.
+        /// Get the document types.
         /// </summary>
+        /// <returns></returns>
         [HttpGet("types")]
-        [HasPermission(Permissions.PropertyAdd)]
-        [ProducesResponseType(typeof(ExternalResult<QueryResult<DocumentType>>), 200)]
-        [SwaggerOperation(Tags = new[] { "documents" })]
+        //[HasPermission(Permissions.DocumentView)]
+        [Produces("application/json")]
+        [ProducesResponseType(typeof(List<Concepts.DocumentTypeModel>), 200)]
+        [SwaggerOperation(Tags = new[] { "types" })]
```

Comment:
types or storage-types or document-types or storage-document-types?

Reply:
should be document types.